src/Authentication.Abstractions/Interfaces/IAzureEnvironment.cs

```diff
@@ -131,5 +131,15 @@ public interface IAzureEnvironment : IExtensibleModel
         /// The set of version profile s(service capabilities) supported
         /// </summary>
         IList<string> VersionProfiles { get; }
+
+        /// <summary>
+        /// The domain name suffix for Azure Attestation tenants
+        /// </summary>
+        string AzureAttestationDnsSuffix { get; set; }
```

Comment:
Adding things to the interface should be fine... have you test it with the code in azure-powershell?

Reply:
I have hard code AttestationDnsSuffix to attest.azure.net in azure-powershell repo to test the data plane operations, it works

Reply:
Let me see if I can build a package of common code for you to test